src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/MaxLatencySpec.groovy

```diff
@@ -242,6 +242,58 @@ class MaxLatencySpec extends HealthCheckSpecification {
         database.resetCosts()
     }
 
+    @Tidy
+    def "Able to create DEGRADED flow with LATENCY strategy if flowPath > max_latency"() {
+        given: "2 non-overlapping paths with 11 and 15 latency"
+        setLatencyForPaths(11, 15)
+
+        when: "Create a flow, maxLatency 5 and maxLatencyTier2 10"
+        def flow = flowHelperV2.randomFlow(switchPair).tap {
+            allocateProtectedPath = false
+            maxLatency = 9
+            maxLatencyTier2 = 16
```

Comment:
can be 12? I don't follow the chosen boundary values. Can you explain?

Reply:
I still don't get it. This should work the same with `maxLatencyTier2 = 12`. Second path with '15' latency is not even used in this test and we are solely focused on the '11' latency path. So if should not satisfy the `maxLatency=10` restriction but should satisfy the tier2 restriction. Is it the case being tested here? If yes, then `16` value completely obfuscates the goal of the test and messes with the second '15' path which is not used in the test

Reply:
Agree, it is not clear. I didn't think from this side.
The system always tries to find a path with the best latency for a flow with the `latency` strategy.
The `max_latency` and `max_latency_tier2` fields are used for setting the status of the flow.

for example:
We have two paths with 11 and 15 latency
also we have two flows (flow_1: mL=10, mlT2=12; flow_2: mL=14, mlT2=16)
Both flows are build via the path with 11 latency.
But the status of the flows are different: flow_1 - degraded, flow_2-up


The `maxLatencyTier2` value in the test was changed to 12, now it should clear.

Reply:
Good example. The code also looks clear now 👍